tchannel/tornado/connection.py

```diff
@@ -283,28 +309,19 @@ def send(self, message):
 
         future = tornado.gen.Future()
         self._outstanding[message.id] = future
-        self.write(message)
+        self._write(message)
         return future
 
-    def write(self, message):
+    def write(self, fragments):
         """Writes the given message up the wire.
 
         Does not expect a response back for the message.
 
-        :param message:
-            Message to write.
+        :param fragments:
+            A generator that will return fragment messages
```

Comment:
This is an API change for TornadoConnection's interface. Are you sure `write` isn't used anywhere else in the library?

If you're really going to change this behavior, also update the docstring and pick a better name for this. Isn't `fragments` just an iterable of messages? Call this `write_all` or something then?